paasta_tools/tron_tools.py

```diff
@@ -266,6 +268,24 @@ def get_env(self):
                     ).values()
                 )
             )
+            if "AWS_ACCESS_KEY_ID" not in env or "AWS_SECRET_ACCESS_KEY" not in env:
+                try:
+                    access_key, secret_key = get_aws_credentials(
+                        service=self.get_service(),
+                        aws_credentials_yaml=self.config_dict.get(
+                            "aws_credentials_yaml"
+                        ),
+                    )
+                    env["AWS_ACCESS_KEY_ID"] = access_key
+                    env["AWS_SECRET_ACCESS_KEY"] = secret_key
+                except Exception:
+                    log.warning(
```

Comment:
Can you also print the exception message? Maybe the yaml was malformed, maybe permission denied, may be file doesn't exist, etc.